core/src/test/scala/org/apache/spark/sql/delta/MergeIntoScalaSuite.scala

```diff
@@ -35,22 +35,29 @@ class MergeIntoScalaSuite extends MergeIntoSuiteBase  with DeltaSQLCommandTest
   import testImplicits._
 
   test("basic scala API") {
-    withTable("source") {
-      append(Seq((1, 10), (2, 20)).toDF("key1", "value1"), Nil)  // target
-      val source = Seq((1, 100), (3, 30)).toDF("key2", "value2")  // source
+    Seq("true", "false").foreach { flag =>
```

Comment:
nit: could you create a new test case instead?

Reply:
Done